lib/services/livesync/livesync-service.ts

```diff
@@ -166,6 +166,7 @@ export class LiveSyncService extends EventEmitter implements IDebugLiveSyncServi
 		} catch (err) {
 			this.$logger.trace("Could not stop application during debug livesync. Will try to restart app instead.", err);
 			if ((err.message || err) === "Could not find developer disk image") {
+				liveSyncResultInfo.isFullSync = true;
```

Comment:
can you set comment here why you are setting isFullSync to true? It does not make any sense to do it.